programs/uxd/src/instructions/credix_lp/collect_profit_of_credix_lp_depository.rs

```diff
@@ -71,7 +71,7 @@ pub struct CollectProfitOfCredixLpDepository<'info> {
 
     /// #8
     #[account(
-        constraint = credix_program_state.credix_multisig_key == credix_multisig.key() @UxdError::InvalidCredixMultisig,
+        constraint = credix_program_state.credix_multisig_key == credix_multisig_key.key() @UxdError::InvalidCredixMultisig,
```

Comment:
Ok here missing that change (the why we use the same name, more idiomatic)
```suggestion
        has_one = credix_multisig_key @UxdError::InvalidCredixMultisig,
```

Reply:
nice catch ser, fixed!